### DISCUSSION TITLE
Shift Print To The Right

### DISCUSSION THREAD

**Discussion by Dakweed666** [mriscoc/Ender3V2S1#1030] [Q&A]
Ender 3 V2 Neo, using mriscoc, OK, can anyone that knows mriscoc tell me why under physical settings I can't set the bed size any higher than 220? Just stops at 220 when you turn the dial. According to others I need to set it to 230, With all my other offsets it prints off to the left hand side. Basically just need to move the print to the right.
![20230816_131037](https://github.com/mriscoc/Ender3V2S1/assets/141825054/49ae8bed-b387-4194-90a4-dbca138d8419)

**Comment by mriscoc** [comment]
> can anyone that knows mriscoc tell me why under physical settings I can't set the bed size any higher than 220

![image](https://github.com/mriscoc/Ender3V2S1/assets/2745567/9db11212-8c2d-41d0-929e-923b78d82896)

https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
